migrate/restore.sh

```diff
@@ -109,6 +109,13 @@ function import_vault() {
     kubectl -n $NAMESPACE exec $VAULT_POD -c vault -- sh -c 'rm -rf /vault/file/sys/expire/id/auth/token/create/* && for k in $(head -n 3 /vault/file/vault-init-out | sed "s/^.*: //g"); do vault operator unseal "$k"; done; vault login $(grep "Root" /vault/file/vault-init-out | sed "s/^.*: //g"); vault token create -period="768h" > /vault/file/client-token; grep "token " /vault/file/client-token | sed "s/^token\W*//g" > /vault/__restricted/client-token'
 }
 
+function reinject_bottoken() {
+    JOB_NAME=$(kubectl -n $NAMESPACE get jobs -o json | jq '.items[0].metadata.name' -r)
```

Comment:
Shouldn't you make sure that you grab the right job here instead of blindly picking the first element in the array? It works right now as we only have that single job but that could change and then this will break.

Reply:
Thats a good point.  After discussions we will be targeting a specific 3.x release to migrate to.  So there will only be this one job and that wont change.